Kitodo/src/main/webapp/WEB-INF/resources/js/shortcuts.js

```diff
@@ -21,6 +21,7 @@ let shortcuts = {
             keyCombination += event.originalEvent.code;
             event.originalEvent.target.value = keyCombination;
             event.preventDefault();
+            $(event.originalEvent.target).change();
```

Comment:
What does this line do?

Reply:
It dispatches a `change` event for the input field. This triggers the primefaces ajax update.